src/Raven.Server/Smuggler/Documents/DatabaseSmuggler.cs

```diff
@@ -483,6 +508,66 @@ private SmugglerProgressBase.Counts ProcessLegacyAttachments(SmugglerResult resu
             return result.Documents;
         }
 
+        private SmugglerProgressBase.Counts ProcessLegacyAttachmentDeletions(SmugglerResult result)
+        {
+            var counts = new SmugglerProgressBase.Counts();
+            using (var actions = _destination.Documents())
+            {
+                foreach (var id in _source.GetLegacyAttachmentDeletions())
+                {
+                    counts.ReadCount++;
+
+                    if (counts.ReadCount % 1000 == 0)
+                    {
+                        var message = $"Read {counts.ReadCount:#,#;;0} legacy attachment deletions.";
+                        result.AddInfo(message);
+                        _onProgress.Invoke(result.Progress);
+                    }
+
+                    try
+                    {
+                        actions.DeleteDocument(id);
+                    }
+                    catch (Exception)
+                    {
+                        counts.ErroredCount++;
+                    }
+                }
+            }
+
+            return counts;
+        }
+
+        private SmugglerProgressBase.Counts ProcessLegacyDocsDeletions(SmugglerResult result)
+        {
+            var counts = new SmugglerProgressBase.Counts();
+            using (var actions = _destination.Documents())
+            {
+                foreach (var id in _source.GetLegacyDocsDeletions())
+                {
+                    counts.ReadCount++;
+
+                    if (counts.ReadCount % 1000 == 0)
+                    {
+                        var message = $"Read {counts.ReadCount:#,#;;0} legacy document deletions.";
+                        result.AddInfo(message);
+                        _onProgress.Invoke(result.Progress);
+                    }
+
+                    try
+                    {
+                        actions.DeleteDocument(id);
+                    }
+                    catch (Exception)
+                    {
+                        counts.ErroredCount++;
```

Comment:
why are you ignoring errors? where are they reported?

Reply:
I will report those errors.